apps/system/utils/netcmd.c

```diff
@@ -89,14 +89,6 @@ extern int iotivity_cloud_main(int argc, char *argv[]);
 #ifdef CONFIG_SYSTEM_NETDB
 extern int netdb_main(int argc, char *argv[]);
 #endif
-#ifdef CONFIG_EXTERNAL_WAKAAMA_EXAMPLE
-#ifdef CONFIG_LWM2M_CLIENT_MODE
-extern int lwm2m_client_main(int argc, char *argv[]);
-#endif
-#ifdef CONFIG_LWM2M_SERVER_MODE
-extern int lwm2m_server_main(int argc, char *argv[]);
-#endif
-#endif
```

Comment:
Yes. You are right.
Platform needs to provide examples or guides for each protocol. So soon we will develop a simple examples for lwm2m client and server (which is supporting UDP/TCP/DTLS/TLS protocols). It will replace "lwm2m_server_main" and "lwm2m_client_main" examples.